packages/core/strapi/tests/api/populate/sanitize.test.e2e.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const { createTestBuilder } = require('../../../../../../test/helpers/builder');
+const { createStrapiInstance } = require('../../../../../../test/helpers/strapi');
+const {
+  createContentAPIRequest,
+  createAuthRequest,
+} = require('../../../../../../test/helpers/request');
+
+const builder = createTestBuilder();
+
+let strapi;
+let rq;
+
+const schemas = {
+  contentTypes: {
+    a: {
+      kind: 'collectionType',
+      displayName: 'a',
+      singularName: 'a',
+      pluralName: 'as',
+      attributes: {
+        cover: { type: 'media', single: true },
```

Comment:
I guess `single: true` works the same way as `multiple: false` in your test?

Reply:
Actually that's a mistake